vxquery-xtest/src/test/resources/ExpectedTestResults/Simple/US000000001.xml

```diff
@@ -0,0 +1,17 @@
+<!--
```

Comment:
No it is not. This is because the original file is not in the same folder as the Expected Results folder. I can change the entire path, if you want. But I feel even then it will not work.